src/models/message.ts

```diff
@@ -0,0 +1,27 @@
+import mongoose from 'mongoose';
+import userRessource from '../models/userRessource';
+
+export interface IMessageModel extends mongoose.Document {
+  receivers: mongoose.Types.DocumentArray<mongoose.Types.Subdocument>;
+}
+
+const messageSchema = new mongoose.Schema({
+  platform: String,
+  template: String,
+  sender: {
+    name: String,
+    mail: String,
+  },
+  payload: [
+    {
+      language: String,
+      payload: Object,
+    },
+  ],
+  receivers: [ userRessource ],
```

Comment:
Scema userRessouce should be declared in this file, since it does not define a model.